infrastructure/kubernetes/app-backend.yml

```diff
@@ -36,6 +36,19 @@ spec:
         env:
         - name: "NODE_ENV"
           value: "production"
+        - name: "API_URL"
+          value: "https://agid-apim-test.azure-api.net/api/v1"
+        - name: "REDIS_URL"
+          value: "redis://redis"
+        - name: "SAML_CALLBACK_URL"
+          value: "https://app-backend.k8s.test.cd.teamdigitale.it/assertionConsumerService"
+        - name: "SAML_ISSUER"
+          value: "https://app-backend.cd.teamdigitale.it"
```

Comment:
the final (production) url will be something like https://app-backend.cd.italia.it

we can use that one for the test environment